### PR TITLE
Fix stringifyRequest in windows when context and request is not in the same driver

### PR DESCRIPTION
**Pull request by sorrycc**
For example:

```
var context = 'C:\foo';
var request = 'E:\bar';

path.relative(context, request);  // -> E:\bar
"./" + path.relative(context, part); // -> ./E:\bar, And this file path is not found
```

### CODE REVIEW

**Comment by sorrycc** [comment]
ping

**Comment by afc163** [comment]
:+1:

**Comment by gbk** [comment]
+1

File: index.js [hunk 1]
```diff
@@ -88,8 +88,14 @@ exports.stringifyRequest = function(loaderContext, request) {
 	var splitted = request.split("!");
 	var context = loaderContext.context || (loaderContext.options && loaderContext.options.context);
 	return JSON.stringify(splitted.map(function(part) {
-		if(/^\/|^[A-Z]:/i.test(part) && context)
-			return "./" + path.relative(context, part).replace(/\\/g, "/");
+		if(/^\/|^[A-Z]:/i.test(part) && context) {
+			part = path.relative(context, part).replace(/\\/g, "/");
```
Comment (sokra) [inline index.js]: Could you move the `replace` to the else case? Absolute windows path should have `\`.

Comment (sorrycc) [inline index.js]: done.

**Comment by sokra** [comment]
Thanks

**Pull request merged**